MedtrumKitUI/ViewModels/Onboarding/DeactivatePatchViewModel.swift

```diff
@@ -35,6 +35,22 @@ class DeactivatePatchViewModel: ObservableObject {
                     return
                 }
                 
+                if let pumpManager = self.pumpManager {
+                    pumpManager.state.previousPatch = PreviousPatch(
```

Comment:
Just curious where do you use this for?

Reply:
Users can check what the state was of their last patch, in case of patch failures. Can be used for debugging/reporting issues.

This is also a feature in the Omnipod driver. But the omnipod "screams", when there is a failure creating stress and people forgetting to check the error codes.